src/methods/sep31Send/sendPayment.ts

```diff
@@ -77,7 +80,7 @@ export const sendPayment = async ({
     );
   }
 
-  const tx = new TransactionBuilder(account, {
+  const tx = new TransactionBuilder(new Account(publicKey, sequence), {
```

Comment:
TransactionBuilder takes different params with the latest `stellar-sdk`

Reply:
Just FYI unfortunately this is actually a [bug](https://github.com/stellar/js-stellar-sdk/issues/653); the next release should resolve this!